synthtool/gcp/templates/python_notebooks_testing_pipeline/.cloud-build/requirements.txt

```diff
@@ -0,0 +1,8 @@
+ipython>=7.0
```

Comment:
similar to above comment, can we pin to exact versions and use renovate to keep up with it? or is that less of an issue with a cloud build script compared to with samples maintenance?

Reply:
I think that's possible, although I've never used `renovate`

Reply:
@leahecole I think my main question here is if this template will be incorporated into repos downstream, wouldn't that mean that the template itself should contain its own `renovate` bot so its usable downstream? And if I do that, will that interfere with the renovate set up for this repo? I can add the requirements to this repo's `requirements.txt` at project root but I'm not sure how that would be passed to all the downstream repos using this template.

Reply:
Good question. Will it be incorporated into samples libraries with owlbot, which already have renovatebot enabled? If so, we would want to modify [this template](https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_library/renovate.json). I believe that adding a new `renovate.json` template would make our 🤖 friend and the folks who maintain it extremely confused.

Or, are these notebook templates going to be used ad-hoc in random repos? If it's going to be used in random repos under the GoogleCloudPlatform and googleapis orgs, we do have renovatebot enabled at the org level and the way to bring it to life would be to create a new `renovate.json` template (like that other one I linked).

Reply:
Other than `python-docs-samples` I'm not sure any of the other repos already have owlbot enabled, though that would be our next step! However, both of the main repos currently are under GCP not googleapis.

Would the suggestion be to add a `renovate.json` to the root of this template, make sure the right file extensions are flagged, and then leave blank (?) requirements files?

Reply:
They all probably have it, but similar to renovate, it can magically be turned on with the addition of its config file (in owlbots case, owlbot.py) 

Given that it sounds like the main repos are in the GCP org, I think that we actually might be covered - renovate will match any `requirements.txt` file now that I think about it, so as long as it's in a repo that has a `renovate.json` we are good!

Reply:
awesome, so I've gone ahead and made sure all dependencies have specific versions and made sure the file names are easy for renovatebot to parse. see latest two commits!